### DISCUSSION TITLE
Add a way to pin scores in game

### DISCUSSION THREAD

**Discussion by BlueChinchompa** [ppy/osu#23211] [Ideas] [answered]
There should be a way to pin scores in game

in stable it could be added right here
![image](https://user-images.githubusercontent.com/130874531/232257969-a99bbd81-aae3-42e3-9d94-d259bb84e08e.png)

idk about lazer but it would be nice to see this at least in stable.

**Comment by nagi-desuuu** [comment, marked as answer]
Stable is feature-locked so you won't see that button there for a while.
That features's on the table for Lazer, though.
